### ISSUE TITLE
Does't work with `declaration-value`

### ISSUE THREAD

**Issue by imkremen** [csstools/stylelint-value-no-unknown-custom-properties#8]
When we have `declaration-value` in `var` function it return false-positive result

Example:
```css
color: var(--my-undefined-color, #ffffff);
```

**Comment by romainmenke** [comment]
This plugin doesn't give warnings on that source code.
If it previously did then this issue was already resolved 🎉
